app/javascript/dashboard/components/widgets/conversation/Message.vue

```diff
@@ -53,22 +53,15 @@
         </span>
         <div v-if="!isPending && hasAttachments">
           <div v-for="attachment in data.attachments" :key="attachment.id">
-            <bubble-image
-              v-if="attachment.file_type === 'image' && !hasImageError"
-              :url="attachment.data_url"
+            <bubble-image-audio-video
+              v-if="
+                attachment.file_type === 'image' ||
+                  attachment.file_type === 'audio' ||
+                  attachment.file_type === 'video'
+              "
```

Comment:
```suggestion
v-if="['image', 'audio', 'video'].includes(attachment.file_type)"
```

Reply:
Can we move this condition to a method since this has more than 2 conditions,w hich would improve readability?